libs/gltfio/src/AssetLoader.cpp

```diff
@@ -982,10 +983,20 @@ bool FAssetLoader::createPrimitive(const cgltf_primitive& inPrim, Primitive* out
                 return false;
             }
 
+            if (!accessor->has_min || !accessor->has_max) {
+                continue;
+            }
+
+            Aabb targetAabb(baseAabb);
             const float* minp = &accessor->min[0];
             const float* maxp = &accessor->max[0];
-            outPrim->aabb.min = min(outPrim->aabb.min, float3(minp[0], minp[1], minp[2]));
-            outPrim->aabb.max = max(outPrim->aabb.max, float3(maxp[0], maxp[1], maxp[2]));
+
+            // We assume that the range of morph target weight is [-1,1].
+            targetAabb.min += float3(minp[0], minp[1], minp[2]);
```

Comment:
Why are we adding the mins here?

Reply:
I guess morph weight could be -1. So model size could increase opposite direction.

Reply:
I don't see how adding the mins makes sense. This can grow the aabb?

Reply:
I didn't check. But this AABB is displacement data (deltas) so it could be I guess. For example base primitive AABB min is [0, 0, 0] and target AABB(for delta) min is [-1, -1, -1] then final AABB min will be [-1, -1, -1].

Reply:
Ah If it's displacement then it makes more sense.